lib/matplotlib/rcsetup.py

```diff
@@ -310,6 +318,25 @@ def validate_colorlist(s):
         msg = "'s' must be of type [ string | list | tuple ]"
         raise ValueError(msg)
 
+def validate_cycler(s):
+    'return a cycler object from a string repr or the object itself'
+    if isinstance(s, six.string_types):
+        try:
+            # TODO: We might want to rethink this...
+            cycler_inst = eval(s, {'cycler': cycler, 'Cycler': Cycler})
```

Comment:
Ahh, of course, because the ``matplotlibrc`` passes in a string...

Could we find a simpler way of passing in a cycle as a string?  Asking the user to pass in code as a param feels bad.  How about:

``
(prop, list) op (prop, list) ...
``
we could use a recursive function to deal with brackets, e.g. ((prop, list) + (prop, list)) * (prop, list)